### DISCUSSION TITLE
FreeSql.Provider.ClickHouse无法安装在.net4.5项目上

### DISCUSSION THREAD

**Discussion by gudufy** [dotnetcore/FreeSql#1213] [General]
严重性	代码	说明	项目	文件	行	禁止显示状态
错误		无法安装程序包“FreeSql.Provider.ClickHouse 3.2.665”。你正在尝试将此程序包安装到目标为“.NETFramework,Version=v4.5”的项目中，但该程序包不包含任何与该框架兼容的程序集引用或内容文件。有关详细信息，请联系程序包作者。

**Comment by 2881099** [comment]
ClickHouse.Client 5.0.3

.NETCoreApp 2.2
[Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http/) (>= 3.1.17)
[NodaTime](https://www.nuget.org/packages/NodaTime/) (>= 3.1.0)
[System.Net.Http](https://www.nuget.org/packages/System.Net.Http/) (>= 4.3.4)
.NETFramework 4.8
[Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http/) (>= 3.1.17)
[NodaTime](https://www.nuget.org/packages/NodaTime/) (>= 3.1.0)
[System.Net.Http](https://www.nuget.org/packages/System.Net.Http/) (>= 4.3.4)
.NETStandard 2.1
[Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http/) (>= 3.1.17)
[NodaTime](https://www.nuget.org/packages/NodaTime/) (>= 3.1.0)
[System.Net.Http](https://www.nuget.org/packages/System.Net.Http/) (>= 4.3.4)

**Reply by luoyunchong** [reply]
也许需要支持一下4.8，现在 只支持了Standard2.1 `<TargetFrameworks>netstandard2.1</TargetFrameworks>`

https://github.com/dotnetcore/FreeSql/blob/master/Providers/FreeSql.Provider.ClickHouse/FreeSql.Provider.ClickHouse.csproj

https://docs.microsoft.com/zh-cn/dotnet/standard/net-standard?tabs=net-standard-2-1


net-standard-2-1 不支持.net framework

**Comment by 2881099** [comment]
需要 https://github.com/DarkWanderer/ClickHouse.Client 支持 .net4.5 才可以
